Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMultiView.css

```diff
@@ -7,81 +7,63 @@ div.tabbedMultiView
   position: relative;
   margin: 0;
   padding: 0;
-  background-color: white;
 }
 
 div.tabbedMultiView > div.wrapper
 {
   position: absolute;
-  background-color: white;
   top: 0;
   bottom: 0;
   left: 0;
   right: 0;
-  width: auto;
-  height: auto;
+  padding-top: var(--remotion-themed-spacing-double);
+  padding-right: var(--remotion-themed-spacing-double);
+  padding-left: var(--remotion-themed-spacing-double);
+
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: max-content max-content auto max-content;
+  grid-template-areas: "topControls" "tabStrip" "view" "bottomControls";
 }
 
 div.tabbedMultiViewActiveView
 {
-  height: 80%;
-  width: inherit;
+  grid-area: view;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
 }
 
 div.tabbedMultiViewActiveView > div.contentBorder
 {
-  position: relative;
-  background-color: white;
-  margin: 0; 
-  padding: 0.5em;
+  padding: var(--remotion-themed-spacing-half);
   overflow: auto;
```

Comment:
please add the scrollbar styleing (see boclist), or, actually, we want to try the 'thin blue" scrollbar used in the popups since this works better with the shadow-effect. quick demo?